lang/ja.js

```diff
@@ -26,13 +26,13 @@ export default {
 	"components.form-element.defaultError": "{label} は無効です。",
 	"components.form-element.defaultFieldLabel": "フィールド",
 	"components.form-element.input.email.typeMismatch": "電子メールが無効です",
-	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
-	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {数値は {max} より小さくなければなりません。} other {数値は {max} 以下でなければなりません。}}",
-	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {数値は {min} より大きくなければなりません。} other {数値は {min} 以上でなければなりません。}}",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {数値は {min} より大きく {max} 未満である必要があります。} other {数値は {min} より大きく {max} 以下である必要があります。}}} other {{maxExclusive, select, true {数値は {min} 以上 {max} 未満である必要があります。} other {数値は {min} 以上 {max} 以下である必要があります。}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {数値は {max} 未満である必要があります。} other {数値は {max} 以下である必要があります。}",
```

Comment:
Missing brace.

Before `The number must be less than {max}` After is same.

Before `The number must be less than or equal to {max}.` After is same